OxfordBodleian/Juel-Jensen/BDLaethf29.xml

```diff
@@ -165,62 +513,345 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                                         <dim type="right">11</dim>
                                         <dim type="left">8</dim>
                                     </dimensions>
+                                    <note>The characters per line are 10–13.</note>
+                                    <ab type="pricking">Ruling is visible; pricking is visible.</ab>
+                                    <ab type="ruling" subtype="pattern">
+                                        <locus from="120r" to="121r"/>Ruling pattern:
+                                        1A-1A/0-0/0-0/C.</ab>
+                                    <ab type="pricking">Primary pricks are visible.</ab>
+                                    <ab type="pricking">Ruling pricks are mostly visible.</ab>
+                                    <ab type="ruling">The upper line is written above the
+                                        ruling.</ab>
+                                    <ab type="ruling">The bottom line is written above the
+                                        ruling.</ab>
+                                    <ab type="ruling"> The ruling pattern is clearly different on
+                                            <locus from="121v" to="123v"/>: three ruled lines above,
+                                        three lines in the middle, and three lines below. The same
+                                        pattern, barely visible, is probably ruled on <locus
+                                            from="1r" to="2v"/>.</ab>
                                 </layout>
                             </layoutDesc>
                         </objectDesc>
-                        
+
                         <handDesc>
-                            <handNote xml:id="h1" script="Ethiopic">
-                           
+                            <handNote xml:id="h1" script="Ethiopic"
+                                corresp="#ms_i1 #ms_i2 #ms_i3 #ms_i4 #ms_i5 #ms_i6">
+                                <locus from="4ra" to="118v"/>
+                                <seg type="ink">Black, red</seg>
+                                <date notBefore="1800" notAfter="1868">first half of the nineteenth
```

Comment:
We briefly discussed on how to indicate the century and we opted for numbers, right? (19th century)